spec/propertybase_id_spec.rb

```diff
@@ -107,28 +101,24 @@
       end
 
       context "non existing object" do
-        let(:input_id) { "zzz456789012345678" }
+        let(:input_id) { "zzz4567890123456" }
```

Comment:
hm, the length was 17 and I just deleted a character at the end so it doesnt matter. But I could have also deleted a `z` from the front